plugins/hls-brittany-plugin/hls-brittany-plugin.cabal

```diff
@@ -17,6 +17,10 @@ extra-source-files:
   test/testdata/**/*.hs
 
 library
+  if impl(ghc >= 9.2)
+    buildable: False
+  else
+    buildable: True
```

Comment:
This is not ideal, afaict, this spreads disabling plugins over multiple files, but it looks like we have a hard time doing better than this.
Is https://github.com/haskell/cabal/pull/7783 good enough to allow us to centralise this again?

Reply:
I don't know whether it's good enough, but I agree that buildabilty with ghc x.y would sit better in the project file

Reply:
I actually think it's better in the plugin files! "Does this plugin work on GHC X" feels like a concern of the plugin.